### PR TITLE
[MODULAR] Buffs the shit out of .50 and adds a bullet rebalance module

### PR DESCRIPTION
**Pull request by Draganfrukts**
## About The Pull Request

Does what it says on the tin.

## Why It's Good For The Game
.50 in its current state is hilariously underpowered, and you can see cases of jacked up people simply brushing two shots off. I mean, fuck man, a .357 is more effective right now. I don't want to live in a world where blind firing with a pistol is more effective than hitting a clean shot with an anti-materiel rifle

Edit: Inherits bullet speed from pull #3816
Edit 2: Add firing delay changes to changelog

## Changelog
balance: After a mean lawsuit, Scarborough Arms is now placing the correct powder load on .50 casings, and are producing them with high quality materials. They are faster and much, much more lethal now!
balance: Sniper rifles have been fitted with heavier plasteel bolts to ensure proper function. Their firing delay has been increased!
tweak: .50 renamed to .50 BMG
/:cl:

### CODE REVIEW

**Comment by Draganfrukts** [comment]
CURSE YOU CI SUITE

**Comment by Vizoplez** [comment]
what the fuck this is cursed no

File: modular_skyrat/modules/bulletrebalance/sniper.dm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/obj/projectile/bullet/p50
+	name =".50 BMG bullet"
+	speed = 0.2
+	damage = 110  //You have 120 health, not a guaranteed instant critical. Unless....
+	paralyze = 1 //Knocks you on your ass hard enough as-is, paralyze super nerfed
+	dismemberment = 60
```
Comment (Vizoplez) [inline modular_skyrat/modules/bulletrebalance/sniper.dm]: no

File: modular_skyrat/modules/bulletrebalance/sniper.dm [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+	armour_penetration = 61 //Bulletproof armor alone will not stop this
+  wound_bonus = 100 //Guaranteed wound, why would it not fuck you up royally?
```
Comment (Vizoplez) [inline modular_skyrat/modules/bulletrebalance/sniper.dm]: no

File: modular_skyrat/modules/bulletrebalance/sniper.dm [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/obj/projectile/bullet/p50
+	name =".50 BMG bullet"
+	speed = 0.2
+	damage = 110  //You have 120 health, not a guaranteed instant critical. Unless....
```
Comment (Vizoplez) [inline modular_skyrat/modules/bulletrebalance/sniper.dm:4]: absolutely fucking not

**Review by Vizoplez — CHANGES_REQUESTED** [review]
Rebalance this shit, 60% dismember can blow your head off through armor, along with 110 damage with a fuckton of AP and guaranteed paralyse, not knockdown, paralyse. Which forces them to drop items. Keep in mind, some space ruin turrets use these, its incredibly poor for gameplay. Kthx
Edit: Furthermore, taking wounds actually does more HP damage over time, or just directly - meaning its very likely if it doesnt dismember, it'll instantly hardcrit you. bruh.

**Comment by Vizoplez** [comment]
image

**Comment by MLGTASTICa** [comment]
Good changes , puts the scary in the .50 cal.

File: modular_skyrat/modules/bulletrebalance/sniper.dm [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+	paralyze = 0 //Knocks you on your ass hard enough as-is
+	dismemberment = 30
+	armour_penetration = 61 //Bulletproof armor alone will not stop this
+  wound_bonus = 95 //Guaranteed wound, why would it not fuck you up royally?
+	var/breakthings = TRUE
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/bulletrebalance/sniper.dm]: What is this

Comment (Draganfrukts) [reply modular_skyrat/modules/bulletrebalance/sniper.dm]: 30% dismemberment chance per shot from 50%
61 armor penetration from 50
95 wound bonus from a whopping 0
var/breakthings unchanged from normal .50 file

Comment (Gandalf2k15) [reply modular_skyrat/modules/bulletrebalance/sniper.dm]: Ah, if the value remains the same just don't include it here. So you can remove all of the unchanged values.

Comment (Draganfrukts) [reply modular_skyrat/modules/bulletrebalance/sniper.dm]: That's all she wrote.

**Comment by Draganfrukts** [comment]
Okay it should be ready now, since the integration test fix was applied on another PR already it shouldn't be an issue if merged

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**